PublicAPI/Microsoft.ApplicationInsights.dll/net45/PublicAPI.Shipped.txt

```diff
@@ -131,6 +132,11 @@ Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.SeverityLevel.get
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.SeverityLevel.set -> void
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.Timestamp.get -> System.DateTimeOffset
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.Timestamp.set -> void
+Microsoft.ApplicationInsights.DataContracts.InnerExceptionCountExceededException
```

Comment:
the rule stated that Exception classes should be public because if they surface to the user code, the user would need to inspect the type.

Reply:
the question than why formatting exception will creep out to the user code? Seems like we need to fix the root cause, not the cosmetics of it

Reply:
If you wish, I can revert this change and save it for a later discussion.

The comment on this class reads: `This exception is used to notify the user that the set of inner exceptions has been trimmed because it exceeded our allowed send limit.` So it was clearly intended to be a user visible exception.

Quote from [CA1064](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1064-exceptions-should-be-public?view=vs-2017)
`An internal exception is only visible inside its own internal scope. After the exception falls outside the internal scope, only the base exception can be used to catch the exception. If the internal exception is inherited from Exception, SystemException, or ApplicationException, the external code will not have sufficient information to know what to do with the exception.`